java-tests/runtime/src/test/kotlin/io/spine/validate/ValidateUntilitySpec.kt

```diff
@@ -80,64 +71,4 @@ internal class ValidateUntilitySpec : UtilityClassTest<Validate>(Validate::class
 
         formatted shouldBe "test 1 test 2"
     }
-
-    @MuteLogging
-    @Nested
-    @DisplayName("test message changes upon `(set_once)` and")
-    internal inner class SetOnce {
```

Comment:
I thought we should preserve them until the deprecated members are removed.

Reply:
I hope we'd finish with Validation fixes and completion soon.